Gemfile.lock

```diff
@@ -215,8 +218,8 @@ GEM
       rainbow (>= 2.2.2, < 4.0)
       ruby-progressbar (~> 1.7)
       unicode-display_width (>= 1.4.0, < 1.6)
-    rubocop-performance (1.3.0)
-      rubocop (>= 0.68.0)
+    rubocop-performance (0.0.1)
```

Comment:
When you add only `bullet` gem you cannot have in `Gemfile.lock` changes for `rubocop-performance` or this two gems are dependent. Please run one more time `bundle` command on this branch and check.